installing/installing-preparing.adoc

```diff
@@ -0,0 +1,86 @@
+[id="installing-preparing"]
+= Selecting a cluster installation method and preparing it for users
+include::modules/common-attributes.adoc[]
+:context: installing-preparing
+
+toc::[]
+
+Before you install {product-title}, think about the following questions to decide what kind of installation process to follow and make sure you that you have all of the resources that you need to prepare your cluster for your users.
+
+[id="installing-preparing-selecting-cluster-type"]
+== Selecting your cluster installation type
+Before you install a {product-title} cluster, you need to select the best installation instructions to follow. Think about your answers to the following questions to select the best option.
+
+[id="installing-preparing-install-manage"]
+=== Do you want to install and manage an {product-title} cluster yourself?
+
+If you want to install and manage {product-title} yourself, you can install it on the following platforms:
+
+* Amazon Web Services (AWS)
+* Microsoft Azure
+* Google Cloud Platform (GCP)
+* {rh-openstack}
+* {rh-virtualization}
+* IBM Z and LinuxONE
+* IBM Power
+* VMware vSphere
+* Bare metal or other platform agnostic infrastructure
+
+You can deploy an {product-title} 4 cluster to both on-premise data centers and to cloud hosting services, but all of the machines in your cluster must be in the same datacenter or cloud hosting service.
+
```

Comment:
This is where the ".Deploying {product-title} by using a managed service" title that I suggested in my previous comment could go.